_includes/v21.1/prod-deployment/multi-region-cache-max-sql-memory.md

```diff
@@ -0,0 +1 @@
+On a multi-region cluster that was deployed without setting `--cache` and `--max-sql-memory`, failure of any node will cause cluster-wide unavailability. These two flags therefore **must** be included when deploying multi-region clusters.
```

Comment:
I'm also working on content related to `max-sql-memory` so this caught my eye. [Our style guide](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#style-and-tone) recommends second person imperative and active voice. Given that, should this be rephrased as follows:

You **must** include these two flags when you deploy multi-region clusters.

Reply:
Thanks! I'm actually closing this PR as it turns out to be unnecessary.